src/components/EmbeddedAskForm.scss

```diff
@@ -103,3 +100,36 @@
   padding: 10px 20px;
   margin: 1em 0;
 }
+
+#MultipleChoice--2 label {
+  width: 100% !important;
+  text-align: center !important;
+}
+
+
+@media only screen and (min-width: 25em) {
+  #MultipleChoice--2 label { 
+    width: initial !important;
+  }
+}
+
+@media only screen and (min-width: 25em) {
+  .form-container header, #ask-form > div > div > div {
```

Comment:
Yah man, I was gonna talk to you about this in the AM. I feel we can improve the markup a lot and make things easier in the CSS side.